package.json

```diff
@@ -60,7 +60,7 @@
     "@angular/platform-browser": "13.2.0",
     "@types/google.maps": "^3.47.3",
     "@types/youtube": "^0.0.46",
-    "material-components-web": "14.0.0-canary.c047f7c19.0",
+    "material-components-web": "14.0.0-canary.9736ddce9.0",
```

Comment:
Not sure what our policy on this is, but should we also update the `packages.bzl` peer dependency? I think we usually do this, but not entirely sure.

Reply:
yeah my bad, I always forget all the places we need to update it